Tests/PostgreSQLTests/PostgreSQLConnectionTests.swift

```diff
@@ -399,9 +399,9 @@ class PostgreSQLConnectionTests: XCTestCase {
     }
 
     func testUnlisten() throws {
-        let completionHandlerExpectation = expectation(description: "notify completion handler called")
-        completionHandlerExpectation.expectedFulfillmentCount = 2
-        completionHandlerExpectation.assertForOverFulfill = true
```

Comment:
@MrMage  do you know of an alternative for these guys on Linux?

Reply:
I am afraid not :-/ I normally solve this by storing all the results provided to the callback handler in a separate array, and once I’ve waited for all expectations, I Test the array‘s contents using regular XCTest methods.